.cargo/config.toml

```diff
@@ -0,0 +1,22 @@
+# General rustflags for all non-wasm32 targets
+[target.'cfg(not(target_arch = "wasm32"))']
+rustflags = ["-C", "link-args=-Wl,-undefined,dynamic_lookup"]
+
+[profile.release.target.wasm32-unknown-unknown]
```

Comment:
My local cargo complains that
```
warning: unused config key `profile.release.target` in `/home/volhovm/code/mina/.cargo/config.toml`
```
Not sure if it's intended or important though.

Reply:
Yes, I want to fix it too. Leaving it open for now.